src/cljs/nnts2/note/components.cljs

```diff
@@ -0,0 +1,47 @@
+(ns nnts2.note.components
+  (:require [re-frame.core :as re-frame]
+            [reagent.core :as r]))
+
+(enable-console-print!)
+
+(def full (r/atom ""))
```

Comment:
Should this be named something else? Might not be easy to gauge what this atom does.

Reply:
removed